manifests/d/DuongDieuPhap/ImageGlass/9.2.0.1208/DuongDieuPhap.ImageGlass.installer.yaml

```diff
@@ -95,10 +95,11 @@ FileExtensions:
 - xbm
 - xpm
 - xv
-Dependencies:
-  PackageDependencies:
-  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
-  - PackageIdentifier: Microsoft.EdgeWebView2Runtime
+#Dependencies:
+#  PackageDependencies:
+#  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
```

When launching the application - 
![image](https://github.com/user-attachments/assets/ee747da3-9669-4e05-b38d-68d14aa05e65)